core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/dot-experiments-configuration.component.html

```diff
@@ -17,14 +17,24 @@
                 pButton
                 type="button"
             ></button>
+            <button
+                class="p-button-outlined"
+                *ngIf="vm.experimentStatus === experimentStatus.SCHEDULED"
+                [loading]="vm.isSaving"
+                (click)="stopScheduleExperiment($event, vm.experiment)"
```

Comment:
Should we rename `stopScheduleExperiment` to `cancelScheduleExperiment`? Stop sounds like it could be started again.

```suggestion
                (click)="cancelScheduleExperiment($event, vm.experiment)"
```